### ISSUE TITLE
What's the relation between CL: cell and PRO: protein

### ISSUE THREAD

**Issue by linikujp** [OmniSearch/ncro#18]
Current design is PRO:protein -BFO2:occurs_in-CL:cell, which is not correct.

**Comment by alanruttenberg** [comment]
Agreed. I will fix. Are you looking at the version in the branch 'superclasses-not-information-entities' ? That's where I've been doing more work.
I guess we can say every cell has_part some protein

**Comment by Huang-OMIT** [comment]
has part was removed earlier. Need to get it back.
